packages/next/src/build/webpack/loaders/next-edge-ssr-loader/render.ts

```diff
@@ -158,12 +158,17 @@ export function getRender({
     event?: NextFetchEvent
   ) {
     const extendedReq = new WebNextRequest(request)
-    const extendedRes = new WebNextResponse()
+    const extendedRes = new WebNextResponse(
+      undefined,
+      // tracking onClose adds overhead, so only do it if `experimental.after` is on.
+      !!process.env.__NEXT_AFTER
+    )
 
     handler(extendedReq, extendedRes)
     const result = await extendedRes.toResponse()
 
     if (event?.waitUntil) {
+      // TODO(after): can we replace this?
```

Comment:
Not clear about this comment, what are we replacing here?

Reply:
Phrased the comment weirdly, the intent was basically this:
```
TODO: remove `internal-edge-wait-until` when internal consumers migrate to `unstable_after()`
```
(it's used by a couple things at vercel)

I can remove this if that's not helpful?

Reply:
replaced with a better TODO